packages/integrations/cloudflare/README.md

```diff
@@ -159,6 +159,30 @@ export function GET({ params }) {
 }
 ```
 
+### `cloudflare.runtime` 
+
+`runtime: "off" | "local" | "remote"`
+default `"off"`
+
+This optional flag enables the Astro dev server to populate environment variables and the Cloudflare Request Object, avoiding the need for Wrangler.
+
+-  `local`: environmental variables are available, but the request object is populated from a static placeholder value. 
+- `remote`: environmental variables and the live, fetched request object are available.
```

Comment:
```suggestion
-  `local`: environment variables are available, but the request object is populated from a static placeholder value. 
- `remote`: environment variables and the live, fetched request object are available.
```
Ignore if "environmental" really is the right word, we just don't use it anywhere else so I'm double checking!

Reply:
Good catch ❤️